Specs/DateTime/Dutch/DateExtractor.json

```diff
@@ -1085,40 +1059,37 @@
   },
   {
     "Input": "Ik ga de zesde zondag terug",
-    "NotSupported": "dotnet",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": [
       {
-        "Text": "zesde zondag",
+        "Text": "zondag",
```

Comment:
Isn't this supported in the English version? If so; I think this case should stay and be marked `NotSupported`.

Reply:
it is supported in the English version, but also there only "sunday" is extracted from "i'll go back sixth sunday"

Reply:
I'll add support for ""sixth Sunday" as an open issue across languages. In the context of this PR, I think the current behaviour is appropriate.